src/Concerns/BaseDataCollectable.php

```diff
@@ -34,11 +35,13 @@ public function count(): int
      */
     public function transform(
         bool $transformValues = true,
+        bool $mapPropertyNames = true,
```

Comment:
@rubenvanassche 
if `behind` means inserting the logic inside wrapExecutionType is kind of possible but very not recommended

Reply:
@rubenvanassche 
you're fine!! 

I did the changes right away after I saw your message, I asked just in case.
I already put the $mapPropertyNames after $wrapExecutionType 18 days ago
Please look at  [files changes](https://github.com/spatie/laravel-data/pull/199/files)